DeploymentCloud/Deployment.Common/deployResources.ps1

```diff
@@ -845,8 +854,20 @@ if($sparkCreation -eq 'y') {
 
     $tokens = Get-Tokens
 	if ($useDatabricks -eq 'n') {
+
+        $sparkTemplate = "Spark-Template.json"
+        $sparkParameter = "Spark-parameter.json"
+
+        $version = ($HDInsightVersion -split '\.')[0]
+        $version = [int]$version
+        if ($version -ge 4 -and $enableHDInsightAutoScaling -eq 'y') {
```

Comment:
Both templates should work for either spark version. Is the version check needed ?

Reply:
I assumed that auto-scaling feature only works for HDInsight 4.0. Is this not true?

Reply:
removed the versioned check as autoscale exists for HDI3.6 as well